embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/session/orchestrator/SessionOrchestratorImpl.kt

```diff
@@ -67,5 +69,6 @@ internal class SessionOrchestratorImpl(
      */
     private fun prepareForNewEnvelope() {
         memoryCleanerService.cleanServicesCollections(internalErrorService)
+        sessionProperties.clearTemporary()
```

Comment:
What happens before this change if a session property is set while the user is in the background? It looks like it would persist in the next session, but with this change, it'll be cleared, which tbh seems like the correct thing to do, but it'll be a behavioural change.

If Eliab is OK with this subtle change/fix, I'm good too.

Reply:
Started discussion in Slack